pkgs/by-name/ta/taler-exchange/package.nix

```diff
@@ -90,6 +92,28 @@ stdenv.mkDerivation (finalAttrs: {
     popd
   '';
 
+  postFixup = ''
+    for file in \
+      taler-config \
+      taler-exchange-helper-measure-test-{form,oauth} \
+      taler-exchange-helper-converter-oauth2-test-full_name \
+      taler-exchange-kyc-aml-pep-trigger.sh \
+      taler-exchange-kyc-kycaid-converter.sh \
+      taler-exchange-kyc-oauth2-{challenger,nda,test-converter}.sh \
+      taler-exchange-kyc-persona-converter.sh \
+      taler-unified-setup.sh \
```

Comment:
This one seems to require a whole lot of other things at runtime (`donau-httpd`, `libeufin-cli`, `taler-merchant-httpd`)...

Is it a test file, or something needed at runtiome? It seems to be some sort of bootstrap/coordinator script, and it's inside the `test` directory [upstream](https://git.taler.net/exchange.git/tree/src/testing/taler-unified-setup.sh) -- perhaps we shouldn't be putting it in `$out/bin`? As it is, it's not going to work.